core/issues.py

```diff
@@ -77,7 +80,10 @@
     {"type": "submission_deleted", "bad": True},
 
     # The submission has not been fully processed yet
-    {"type": "submission_processing", "bad": False}
+    {"type": "submission_processing", "bad": False},
+
+    # The submission is linked to a NSFW post
+    {"type": "submission_is_NSFW", "bad": True}
```

Comment:
During the rebase, make sure this issue is the last one in the list so that resynchronizing the issues with the database goes smoothly